class-two-factor-core.php

```diff
@@ -737,15 +748,21 @@ public static function login_url( $params = array(), $scheme = 'login' ) {
 	 * @return array
 	 */
 	public static function create_login_nonce( $user_id ) {
-		$login_nonce = array();
+		$login_nonce = array(
+			'expiration' => time() + HOUR_IN_SECONDS,
```

Comment:
Do we really need a nonce that is valid for an hour? How about we reduce it to 10 minutes or something like that?

Reply:
Addressed in #473 👍🏻